marconi-chain-index/src/Marconi/ChainIndex/Indexers/MintBurn.hs

```diff
@@ -132,6 +133,46 @@ fromAlonzoData = C.fromPlutusData . LA.getPlutusData -- from cardano-api:src/Car
 
 -- * Sqlite
 
+data TxMintRow = TxMintRow
+    { _txMintRowSlotNo          :: !C.SlotNo
```

Comment:
Ah just remembered why we don't want this. That's because `ChainPoint` has a data constructor which has no parameters (ChainPointAtGenesis). In our scenario, we don't want to represent ChainPointAtGenesis in JSON.